### ISSUE TITLE
Keyboard focus is wrong

### ISSUE THREAD

**Issue by smurfix** [ozmartian/vidcutter#315]
Linux, Debian Bullseye.

* open a  video
* observe the Space key work correctly
* click the 'i' button
* dismiss the resulting dialog(s)
* press the space bar
* observe the system behave as if the 'i' button has been clicked instead of playing video

This happens with most buttons. I need to click on the Play button with the mouse to return to reasonable behavior.

**Comment by ozmartian** [comment]
The "i" button shouldn't be responding as its running in non-interactive mode. Must be a Debian dependency issue with the way mpv is built as I cannot replicate this in Arch Linux, Ubuntu, openSUSE or Fedora.

**Comment by smurfix** [comment]
I mean the 'i' in the bottom right of the window, not the key on the keyboard.

**Comment by smurfix** [comment]
Related to this, as soon as I click on a clip in the Clip index the keyboard focus is on the clip; then, cursor up/down switches between clips instead of controlling the current video and the space bar doesn't work any more.

The fix for all of this is to click the timeline; alternately I can lose the focus by clicking on Play/Pause twice and then pressing RightArrow five times, which causes the focus to be lost and the video is stepped forward.
